k8s/upgrade/src/upgrade_resources/upgrade.rs

```diff
@@ -97,6 +98,10 @@ pub struct UpgradeArgs {
     /// Upgrade to the specified branch.
     #[clap(global = true, hide = true, long, short = 'u')]
     pub upgrade_to_branch: Option<String>,
+
+    /// Upgrade tag for testing, contains timestamp.
```

Comment:
nit: The testing use-case is of no relevance to this codebase. The doc comment should talk about what this option does for this codebase, i.e. overrides the container image tag for the upgrade job pod.